<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modbus</class>
 <widget class="QWidget" name="modbus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="logOnlyWrite">
     <property name="text">
      <string>Log Only Write command</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButtonNewDevice">
     <property name="text">
      <string>New device</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButtonLoadSetup">
     <property name="text">
      <string>Load setup</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>M3 Mux Setup</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QSpinBox" name="MuxInd">
        <property name="suffix">
         <string/>
        </property>
        <property name="prefix">
         <string>Index Register : </string>
        </property>
        <property name="minimum">
         <number>12</number>
        </property>
        <property name="maximum">
         <number>27</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSpinBox" name="MuxVal">
        <property name="suffix">
         <string/>
        </property>
        <property name="prefix">
         <string>Value Register : </string>
        </property>
        <property name="minimum">
         <number>12</number>
        </property>
        <property name="maximum">
         <number>27</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Protocol</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="ButtonRTU">
        <property name="text">
         <string>RTU-through TCP</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="Modbus_TCP_Enable">
        <property name="text">
         <string>Modbus-TCP</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="clearLog">
     <property name="text">
      <string>Clear Log</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButtonSaveSetup">
     <property name="text">
      <string>Save setup</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
